serving/src/main/java/ai/djl/serving/http/InferenceRequestHandler.java

```diff
@@ -87,7 +87,7 @@ protected void handleRequest(
         switch (segments[1]) {
             case "ping":
                 ModelManager.getInstance()
-                        .workerStatus()
+                        .workerStatus(TYPES[1])
```

Comment:
We should implement a KServeRequestHandler, see InvalidRequestHandler.
You can register KServeRequestHandler to both management and inference channel pipeline. 

Should all KServe API all on Management port?